ci/gpu/build.sh

```diff
@@ -63,6 +63,7 @@ if hasArg --skip-tests; then
 else
     logger "Installing extra test dependencies..."
     conda install "cupy>=6.0.0"
+    conda install "numba>=0.49.0"
```

Comment:
This should be on the same line as the other conda install. It will save time solving the environment.

Reply:
@dillon-cullinan this is just temporary as we bump the gpuci-build-environment Dockerfile anyway 😄. We can likely remove both installs after that!